thirdeye/thirdeye-frontend/app/pods/manage/alert/tune/route.js

```diff
@@ -277,7 +284,7 @@ export default Route.extend({
 
       fetch(tuneIdUrl + configString, postProps('')).then(checkStatus)
         .then((autoTuneId) => {
-          return Ember.RSVP.hash(tuningPromiseHash(startDate, endDate, autoTuneId[0], alertId));
+          return Ember.RSVP.hash(tuningPromiseHash(startDate, endDate, autoTuneId[0], alertId, severityVal));
```

Comment:
can autoTuneId be an empty array here?

Reply:
It comes in as an array of one. If empty, fetch will catch an error below.